proto/ibc/applications/transfer/v1/tx.proto

```diff
@@ -49,6 +49,8 @@ message MsgTransfer {
   uint64 timeout_timestamp = 7;
   // optional memo
   string memo = 8;
+  // tokens to be transferred
+  repeated cosmos.base.v1beta1.Coin tokens = 9 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
```

Comment:
should we deprecate the `token` field?

Reply:
I think it could be removed on the `token` potentially. This would need to be tested with ledger signing for compatability. If we deprecate and leave the tag, an empty `Coin` would always be displayed on ledger screen before signing (don't omit empty) but if we remove the tag and the `Coin` is empty because the new repeated field `tokens` is being used then I would imagine the empty coin is omitted and not displayed on the ledger screen (which I think is better ux).

Reply:
Aside from above. The repeated field needs: `(amino.encoding) = "legacy_coins",`

Reply:
sdk also uses `(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"` for fields like this, maybe we should also be doing that. 

ref: https://github.com/cosmos/cosmos-sdk/blob/e8f28bf5db18b8d6b7e0d94b542ce4cf48fed9d6/proto/cosmos/bank/v1beta1/genesis.proto#L23-L26